admin_manual/configuration_files/external_storage_configuration_gui.rst

```diff
@@ -131,6 +131,19 @@ on your network!
 .. figure:: external_storage/images/user_mounts.png
    :alt: Checkboxes to allow users to mount external storage services.
 
+Setting Up Google Drive and Dropbox Connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When an external storage is created which uses either Google Drive or Dropbox, a link to the respective configuration page is available, next to the service name. 
+
+.. figure:: external_storage/images/external-storage-google-drive-and-dropbox-configuration.png
+   :alt: Links to Google Drive and Dropbox app configuration pages.
+
+In the screenshot above, you can see that two external storage connections have been created, but not configured. 
+One goes to Google Drive, the other to Dropbox. 
+If you click the cog icon next to the name of either, the respective app configuration page will open in a new tab, or a new window. 
+From there, you can manage the configuration and obtain the respective credentials needed for configuring the connection.
```

Comment:
I just tried with Dropbox, and there are a few steps to take that are "non-obvious" to mere mortals. The user has to click a button to "Create App" and then among other items to choose from, I had to make up an "App Name". It is not really obvious that I am just "enabling a file-sharing API interface to Dropbox".
I guess it is not OC's job to detail how to drive these Dropbox pages, but how do users make the connection successfully when they do not really know what they are meant to be doing in the Dropbox UI?